src/ChannelServer/Util/GmCommands.cs

```diff
@@ -42,7 +42,7 @@ public GmCommands()
 			Add("jump", "<x> <y> <z>", HandleJump);
 			Add("warp", "<map id> <x> <y> <z>", HandleWarp);
 			Add("item", "<item id> [amount]", HandleItem);
-			Add("spawn", "<monster id>", HandleSpawn);
+			Add("spawn", "<monster id> <x> <y> <z>", HandleSpawn);
```

Comment:
What do you believe we can achieve by letting users specify a position?

The command manager currently can only handle indexed parameters, which makes them a rare commodity, because each new one has an impact on the usability and complicates the parsing. I'd suppose there aren't many who have a need to spawn a monster at a specific position, usually you'd want to spawn it where *you* are, or at the position of someone else, both of which can be achieved without a position. In my opinion there are better uses for the parameters.

On a side-note, the parameters you added are optional, you should use squared brackets to document that, see the item command for an example.